src/brain/issueLabelHandler/index.test.ts

```diff
@@ -313,6 +336,8 @@ describe('issueLabelHandler', function () {
       await addLabel('Product Area: Does Not Exist', 'sentry-docs');
       expectUntriaged();
```

Comment:
Not too big a fan of using these functions, since I'm finding myself scrolling up to see what labels are expected with `expectUntriaged` and `expectRouted`. @chadwhitacre thoughts on removing this and just explicitly asserting labels such as below?

Reply:
From call ... I'm fine with refactoring if you want to.